webclient/src/index.html

```diff
@@ -278,7 +318,7 @@
 
     <!-- Feedback modal -->
     <div class="modal" id="feedback-modal">
-      <a id="feedback-overlay" class="modal-overlay" aria-label="Close"></a>
+      <button id="feedback-overlay" class="modal-overlay" aria-label="Close"></button>
```

Comment:
```suggestion
      <div id="feedback-overlay" class="modal-overlay"></div>
```
This is actually not really a button, it's just the transparent background overlay that you can click on to make the dialog close. If it irritates Google I'd suggest using a `div` here, since a `button` breaks in Firefox for me.

Also, I'd remove the `aria-label` on this one then, because there is already a second (and real) close button below, and for screenreaders two buttons with the same text are probably not helpful.